src/libs/GooglePlacesUtils.js

```diff
@@ -10,18 +10,24 @@ import _ from 'underscore';
  * }
  *
  * @param {Array} addressComponents
- * @param {String} type
- * @param {String} key
- * @returns {String|undefined}
+ * @param {Object} fieldsToExtract – has shape: {addressType: 'keyToUse'}
```

Comment:
This does not look like what we are passing in from AddressSearch.js...
```js
{
            street_number: 'long_name',
            route: 'long_name',
            locality: 'long_name',
            sublocality: 'long_name',
            postal_code: 'long_name',
            administrative_area_level_1: 'short_name',
}
```

Reply:
Spoke about this 1:1. It appears the documentation for the shape of `fieldsToExtract` isn't correct (the object should be wrapped in an array). Tested it after that and it worked.

Reply:
updated the comment above, `fieldToExtract` should be right, right?